docker-cloudwatch-exporter.tf

```diff
@@ -0,0 +1,66 @@
+resource "github_repository" "cloudwatch_exporter" {
+  name             = "cloudwatch-exporter"
+  description      = "cloudwatch-exporter"
```

Comment:
can you give it a proper description?

Reply:
Sorry hadn't raised it yet, was still reviewing myself, but of course sure.